File: alumni_app/src/app/data/model/Survey.ts
import { Question } from "./Question";
export class Survey {
    id?: number;
    title: string;
    description: string;
    questions: Question[];
    estado?: boolean = false;
    constructor(title: string, description: string,estado:boolean, questions: Question[] = []) {
        this.title = title;
        this.description = description;
        this.questions = questions.slice(); 
        this.estado = estado;
    }
}   

File: alumni-back/src/main/java/ec/edu/ista/springgc1/model/entity/Survey.java
package ec.edu.ista.springgc1.model.entity;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

import javax.persistence.*;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@Entity
@Data
public class Survey {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String title;

    private String description;
    @OneToMany(mappedBy = "survey", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
    @JsonIgnoreProperties("survey")
    private List<Question> questions;
    private Boolean estado;
}
